docs/products/kafka/kafka-connect/howto/manage-connector-versions.md

```diff
@@ -69,6 +72,16 @@ as part of an Aiven for Apache Kafka service, use the [Aiven API](https://api.ai
 [Aiven Provider for Terraform](https://registry.terraform.io/providers/aiven/aiven/latest/docs)
 to set the connector version.
 
+:::warning
+Although the version selector appears under the configuration page of connector instances in
+Aiven Console for convenience, the actual choice affects all instances of the connector under
+the same service. It is not limited to the scope of a single connector instance.
+
+Prior to [KIP-891](https://cwiki.apache.org/confluence/display/KAFKA/KIP-891%3A+Running+multiple+versions+of+Connector+plugins)
+which will presumably get introduced with Apache Kafka 4.1.0, it is not possible for a Connect
+worker to load multiple versions of the same plugin at the same time. It means that if you run
+multiple instances of a connector in a given service, they must rely on the same library version.
```

Comment:
```suggestion
The version selector in the Aiven Console appears on individual connector configuration pages, 
but changing the version affects all instances of that connector within the service, 
not just the selected instance.

Apache Kafka Connect does not support loading multiple versions of the same plugin within a service. 
This limitation is expected to be addressed in Apache Kafka 4.1.0 through [KIP-891](https://cwiki.apache.org/confluence/display/KAFKA/KIP-891%3A+Running+multiple+versions+of+Connector+plugins). 
Until then, if you run multiple instances of a connector in a service, they 
must all use the same plugin version.
```

Rephrased to align with editorial guidelines and improve clarity. Avoids technical jargon and simplifies phrasing for broader readability.